tests/support/a11y.ts

```diff
@@ -0,0 +1,100 @@
+import { Result } from 'axe-core';
```

Comment:
Is there any technical reason to put this Cypress-specific code here? Cannot it go to `integration-tests`?

Reply:
I put all dependent files in a separate test folder for better segregation